sdks/python/apache_beam/dataframe/partitionings.py

```diff
@@ -115,6 +115,23 @@ def partition_fn(self, df, num_partitions):
     for key in range(num_partitions):
       yield key, df[hashes % num_partitions == key]
 
+  def check(self, dfs):
```

Comment:
This isn't a very strong check if the dataframes are fairly sparse (as they are in most examples). We could try concat + repartition and verify the results are the same.

Reply:
Added a TODO to do this in a later PR